config/environments/development.rb

```diff
@@ -115,6 +115,11 @@
   REPOSITORY_STORAGE = "#{::Rails.root.to_s}/data/dev/repos"
 
   ###################################################################
+  # Directory where Repositories will be created. Make sure MarkUs is allowed
+  # to write to this directory
+  REPOSITORY_STORAGE_GITOLITE = "#{::Rails.root.to_s}/data/dev/repos/gitoliteRepos"
```

Comment:
I don't agree with this one. This particular file should be exempt from styling rules to be more human-readable. @david-yz-liu